ambry-cloud/src/main/java/com/github/ambry/cloud/VcrServer.java

```diff
@@ -208,6 +226,32 @@ public void startup() throws InstantiationException {
 
       networkServer.start();
 
+      // Start netty http2 server
+      if (currentNode.hasHttp2Port()) {
```

Comment:
This part is shared between AmbryServer and VcrServer. Can we separate this out in an util method that both AmbryServer and VcrServer can share?

Reply:
I tried to created a common util for them to share. but the it requires many parameters(almost all variables). It not elegant unfortunately.